hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -279,6 +279,12 @@ private FlinkOptions() {
       .defaultValue(128)
       .withDescription("Max log block size in MB for log file, default 128MB");
 
+  public static final ConfigOption<Integer> WRITE_LOG_MAX_SIZE = ConfigOptions
```

Comment:
Can we add the unit like other config options?

Reply:
Let's use option without units, no one confuses about that and it is more straight-forward.

Reply:
If you think it's necessary to add a unit, can we remove all the `.MB` so that we can keep consistance？

Reply:
We can, would remove the `.MB` options in another PR.

Reply:
OK, please file a Jira ticket to track the work.

Reply:
see https://issues.apache.org/jira/browse/HUDI-1849